_docs/example-collections.md

```diff
@@ -926,8 +933,30 @@ headers.Add("Authorization-Key", "<key from the webhook headers>");
 if (configuration.ValidWebhookRequest(url, webhookContent, headers))
 {
     Webhook webhook = configuration.ParseString<Webhook>(webhookContent);
-    if (webhook.Event.StartsWith("transaction")) {
-        TransactionWebhook transactionWebhook = configuration.ParseString<TransactionWebhook>(webhookContent);
+    if (webhook.Event.StartsWith("payin_method")) {
```

Comment:
we should only cover the important ones in the example, not all, similarly to how the previous example only covered the most important ones. Do note, that those webhooks DO cover most cases, so we would only likely need the extra payin_method webhooks for error handling, e.g. knowing when to ask for a retry